runtime/src/main/java/io/quarkiverse/fx/RunOnFxThreadInterceptor.java

```diff
@@ -13,10 +16,18 @@
 public class RunOnFxThreadInterceptor {
 
     private static final Logger LOGGER = Logger.getLogger(RunOnFxThreadInterceptor.class);
+    // The startup latch from FxApplication
+    @PrimaryStage
+    @Inject
+    CountDownLatch hasStarted;
```

Comment:
I would have named it `startupLatch` like in your comment since it's not a boolean

Reply:
Changed.